ledger/participant-integration-api/src/test/lib/scala/platform/store/dao/JdbcLedgerDaoPartiesSpec.scala

```diff
@@ -143,21 +144,6 @@ private[dao] trait JdbcLedgerDaoPartiesSpec {
     }
   }
 
-  it should "inform the caller if they try to write a duplicate party" in {
```

Comment:
Shouldn't this test be retained for old schema?

Reply:
oh, yeah! Re-added those with conditions: they will be only running for mutable schema!